h/migrations/versions/ff02f2c58_add_the_organization_table.py

```diff
@@ -0,0 +1,28 @@
+"""
+Add the organization table.
+
+Revision ID: ff02f2c58
+Revises: 7bf167611bc3
+Create Date: 2018-03-15 11:00:50.420618
+"""
+
+from __future__ import unicode_literals
+
+from alembic import op
+import sqlalchemy as sa
+
+revision = 'ff02f2c58'
+down_revision = '7bf167611bc3'
+
+
+def upgrade():
+    op.create_table(
+        'organization',
+        sa.Column('id', sa.Integer, autoincrement=True, primary_key=True),
+        sa.Column('name', sa.UnicodeText, nullable=False, index=True),
+        sa.Column('logo', sa.UnicodeText),
```

Comment:
I think we need to add an `authority` column to the `organization` table as well, the same as the `authority` column that users and groups have. Because I think we need to make sure that all of the groups that belong to a given organization all have the same authority (the same authority as the organization). I think we probably want to rule out the situation where a single organization can contain groups from different authorities, at least for now.

Can you add `authority = sa.Column(sa.UnicodeText(), nullable=False)`?